lib/shared/thread-utils.js

```diff
@@ -52,10 +52,23 @@ function viewerIsMember(threadInfo: ?(ThreadInfo | RawThreadInfo)) {
   );
 }
 
+function threadIsInHome(threadInfo: ?(ThreadInfo | RawThreadInfo)) {
+  return threadInfo && threadInfo.currentUser.subscription.home;
+}
+
 function threadInChatList(threadInfo: ?(ThreadInfo | RawThreadInfo)) {
   return (
     viewerIsMember(threadInfo) &&
-    threadHasPermission(threadInfo, threadPermissions.VISIBLE)
+    threadHasPermission(threadInfo, threadPermissions.VISIBLE) &&
+    threadIsInHome(threadInfo)
+  );
```

Comment:
Because of this (for now at least) in web version only visible chats are those not in background

Reply:
Sounds good, for now on web it is like before